ci/build_python.sh

```diff
@@ -21,51 +19,88 @@ CPP_CHANNEL=$(rapids-download-conda-from-s3 cpp)
 
 sccache --zero-stats
 
-# TODO: Remove `--no-test` flag once importing on a CPU
-# node works correctly
-# With boa installed conda build forwards to the boa builder
 
-RAPIDS_PACKAGE_VERSION=$(head -1 ./VERSION) rapids-conda-retry build \
-  --no-test \
-  --channel "${CPP_CHANNEL}" \
-  conda/recipes/pylibcudf
+RAPIDS_PACKAGE_VERSION=$(head -1 ./VERSION)
+export RAPIDS_PACKAGE_VERSION
+
+# populates `RATTLER_CHANNELS` array
+source rapids-rattler-channel-string
+
+rapids-logger "Prepending channel ${CPP_CHANNEL} to RATTLER_CHANNELS"
```

Comment:
Follow-up request: please separate each package build with a call to `rapids-logger` like `rapids-logger "Building pylibcudf"`, etc.

Reply:
done